src/config/tokens.json

```diff
@@ -1,4 +1,5 @@
 {
   "openmaptiles": "get_your_own_OpIi9ZULNHzrESv6T2vL",
-  "thunderforest": "b71f7f0ba4064f5eb9e903859a9cf5c6"
+  "thunderforest": "b71f7f0ba4064f5eb9e903859a9cf5c6",
+  "locationiq": "pk.put_your_api_key_here7bb23dffeb4"
```

Comment:
When I test this PR, this key does not seem to be authorised to load a map.
Is this intentional?

Reply:
It has HTTP referrer restrictions set to work for maputnik & localhost domains, are you using a different hostname in your environment?

Reply:
Works like a charm 👍🏻